### DISCUSSION TITLE
Lettuce behavior when redis db is down

### DISCUSSION THREAD

**Discussion by codingconquerors** [redis/lettuce#2364] [Q&A]
Hello Friends,

Does lettuce handles things gracefully when redis db goes down suddenly.? What I mean is I do not have to implement circuit breaker pattern explicitly for this purpose.
Actually, I do not want to consume my connections from connection pools in case redis db goes down suddenly.

Any help would be appreciated.

Thanks.

**Comment by mp911de** [comment]
Check out `DisconnectedBehavior` in the [reference documentation](https://lettuce.io/core/release/reference/index.html#client-options).
